### ISSUE TITLE
rfc: move from Clickhouse

### ISSUE THREAD

**Issue by panariga** [Swetrix/swetrix#357]
### Dupe Check

- [X] I have [searched Swetrix feature requests](https://github.com/swetrix/swetrix/issues?q=label%3Aenhancement) and there are no duplicates

### Describe the solution you'd like?

regular SQL compability to work with MySQL and MariaDB

### Additional context

Is there any chance to get rid of ClickHouse for self-hosting? 
I appreciate your work, but for small and medium projects ClickHouse is complete BS. 
It eats 5x more CPU time than a MariaDB instance on my PrestaShop. 
Each product page request involves over 200+ MariaDB requests and one to save stats for Swetrix. 
But still, ClickHouse consumes energy for nothing. 
Maybe it's nothing for big tech, but for SMBs that want to keep their data safe, Swetrix is a perfect destination. 
![Screenshot 2024-12-13 135600](https://github.com/user-attachments/assets/4d987a92-857b-4727-a1eb-79375f714f1e)
However, this useless hardware waste is unforgivable.

### How important is this feature to you?

3

**Comment by Blaumaus** [comment]
I don't think this is something that can be done soon as Swetrix already relies on Clickhouse too much. But as an alternative, I released some configs to help preserve the RAM - https://github.com/Swetrix/selfhosting/tree/main/clickhouse. They can be plugged in if needed in the `compose.yaml` file

**Comment by Blaumaus** [comment]
I'll close it for now. Migrating from Clickhouse is too much work really and I'm not sure if it's worth doing at the moment. Any suggestions for optimising Clickhouse performance on the Swetrix Community Edition are welcome!
